src/PortfolioPlanning/Components/Plan/DependencyPanel.scss

```diff
@@ -34,6 +34,7 @@
         display: flex;
         cursor: pointer;
 
+        .info-icon,

```

Comment:
It looks like the icon is slightly off center, any easy fix for that?

Reply:
seems to be a little tricky, have tried to add `align-items: center` for the parent element, but have not worked, need to try more things.

Reply:
This is now fixed.